lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -1144,12 +1144,59 @@ function dosomething_reportback_get_reportbacks($uid = NULL) {
     global $user;
     $uid = $user->uid;
   }
-  return db_select('dosomething_reportback', 'rb')
-    ->fields('rb', array('rbid'))
-    ->condition('uid', $uid)
-    ->orderBy('created', 'DESC')
-    ->execute()
-    ->fetchCol(0);
+
+  // If Rogue is on, return array of themed reportbacks as done in https://github.com/DoSomething/phoenix/blob/76164b2006fb5a266e8166688472742556eae33e/lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_user.inc#L32-L51 for user profile page.
+  // This will be okay because this function is only called in dosomething_user.theme.inc for user profile page.
+  if (variable_get('rogue_collection', FALSE)) {
+    $northstar_id = dosomething_user_get_northstar_id($uid);
+
+    $reportback_items = [];
+
+    $response = dosomething_rogue_get_reportback_items([
+      'as_user' => $northstar_id,
+    ]);
+
+    foreach ($response['data'] as $delta => $reportback) {
+      $reportback = (object) $reportback;
+
+      $total_kudos = $reportback->kudos['data'][0]['term']['total'];
```

Comment:
@katiecrane yes the documentation is out of date (I have a note to update this in my notebook!)